trollhunter-unity/Assets/Scripts/TrollBehavior.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class TrollBehavior : MonoBehaviour {
+
+	public static readonly List<long> LevelToExperience = new List<long> {0, 100, 200, 300, 400};
+
+	public enum LifeCycle {
+		EGG,
+		BABY,
+		ADULT,
+	}
+
+	public struct InternalParameter {
+		public long experience;
+	}
+
+	private LifeCycle lifeCycle;
+
+	private InternalParameter parameter;

```

Comment:
Again here, parameter is really vague.  Stats or something would be a better name.